server/middleware/populateJourney.ts

```diff
@@ -25,12 +26,14 @@ export default function populateJourney(): RequestHandler {
       Object.defineProperty(req.session, p, {
         get() {
           // Will return either the found, mapped, non-undefined session data journey or null
-          return req.session.sessionDataMap[req.params.journeyId]?.[p] ?? null
+          const journeyId = req.params.journeyId ?? 'default'
+          return req.session.sessionDataMap[journeyId]?.[p] ?? null
```

Comment:
I've added this as a way to allow the `journeyMetrics` session use journey ids on journeys which support it while still working on journeys which don't (like create activity). This means journey metrics will no longer be corrupted if a user starts multiple journeys so long as that journey supports journey ids.